### PR TITLE
organizing pieces

### PR DESCRIPTION
**Pull request by giupb**

### CODE REVIEW

File: pieces/AudioTranscriptionPiece/piece.py [hunk 1]
```diff
@@ -52,22 +52,28 @@ def piece_function(self, input_model: InputModel):
...
+                message=msg,
+                string_transcription_result=full_transcript
+            )
+                
+        output_file_path = f"{self.results_path}/{input_model.output_file_name}"
+        with open(output_file_path, "w") as f:
+            f.write(full_transcript)
 
-            output_file_path = f"{dir_path}/{file_name}_transcription_result.txt"
+        # Finally, results should return as an Output model
```
Comment (luiztauffer) [inline pieces/AudioTranscriptionPiece/piece.py:64]:
```suggestion
```

File: pieces/PromptForImageGeneratorPiece/models.py [hunk 2]
```diff
@@ -1,6 +1,13 @@
-from pydantic import BaseModel, Field
+from pydantic import BaseModel, Field, FilePath
 from enum import Enum
```
Comment (luiztauffer) [inline pieces/PromptForImageGeneratorPiece/models.py:2]:
```suggestion
from enum import Enum


```

File: pieces/TextSummarizerPiece/piece.py [hunk 3]
```diff
@@ -1,11 +1,19 @@
 from domino.base_piece import BasePiece
 from .models import InputModel, OutputModel
 from typing import List
+from enum import Enum
 import openai
 import tiktoken
 import asyncio
 
 
+class TokenLimit(int, Enum):
```
Comment (giupb) [inline pieces/TextSummarizerPiece/piece.py:10]: @luiztauffer, I've created this class to map the token limit for each OpenAI model. If you have any better ideas, please let me know.

Comment (luiztauffer) [reply pieces/TextSummarizerPiece/piece.py:10]: looks good to me

**Pull request merged**